server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/ThreadGetContract.scala

```diff
@@ -160,16 +210,24 @@ trait ThreadGetContract {
     assertThatJson(response)
       .isEqualTo(
         s"""{
-          |    "sessionState": "${SESSION_STATE.value}",
-          |    "methodResponses": [
-          |        [
-          |            "error",
-          |            {
-          |                "type": "accountNotFound"
-          |            },
-          |            "c1"
-          |        ]
-          |    ]
-          |}""".stripMargin)
+           |	"sessionState": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
+           |	"methodResponses": [
+           |		[
+           |			"Thread/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"state": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
+           |				"list": [{
+           |					"id": "$threadId",
+           |					"emailIds": ["$message1Id", "$message2Id"]
+           |				}],
+           |				"notFound": [
+           |
+           |				]
+           |			},
+           |			"c1"
+           |		]
+           |	]
+           |}""".stripMargin)
   }
 }
```

Comment:
- In-Reply-To references messsage1 but different subject -> NOMATCH
 - REFERENCES -> match
  -Maybe 1-2 other fonctional tests.